docs/install/install_everest_helm_charts.md

```diff
@@ -0,0 +1,174 @@
+                
+        ```sh
+        kubectl port-forward svc/everest 8080:8080 -n everest-system
+        ``` 
+
+        To launch the Percona Everest UI and create your first database cluster, go to your localhost IP address [http://127.0.0.1:8080](http://127.0.0.1:8080).
+
+
+5. The default username for logging into the Everest UI is `admin`. You can set a different default admin password by using the `server.initialAdminPassword` parameter during installation.
+
+    The default admin password is stored in plain text. It is highly recommended to update the password using `everestctl` to ensure that the passwords are hashed.
+
+5. Deploy additional database namespaces:
+
+    Once Everest is successfully running, you can create additional database namespaces using the `everest-db-namespace` Helm chart. 
+
+    If you set `dbNamespaces.enabled=false` in the previous step, you can deploy a database namespace with the following command:
+
+    ```sh
+    helm install everest \
+    percona/everest-db-namespace \
+    --create-namespace \
+    --namespace everest
+    ```
+
+    !!! note
+        -  All database operators are installed in your database namespace by default. You can override this by specifying one or more of the following options: `[dbNamespace.pxc=false, dbNamespace.pg=false, dbNamespace.psmdb=false]`.
+        - Installation without chart hooks (i.e, the use of `--no-hooks`) is currently not supported.
+
+
+
+
+
+## Configure parameters
+
+You can customize various parameters in the Percona Everest Helm charts for your deployment to meet your specific needs. Refer to the [Helm documentation](https://helm.sh/docs/chart_best_practices/values/){:target="_blank"} to discover how to configure these parameters.
+
+A few parameters are listed in the following table. For a detailed list of the parameters, see the [README](https://github.com/percona/percona-helm-charts/blob/main/charts/everest/README.md#configuration){:target="_blank"}.
+
+|**Charts**|**Key**|**Type**|**Default**|**Description**|
+|---------|------|---------|------------------|------------|
+|everest|`server.initialAdminPassword`|string|""|Initial password configured for admin user.</br></br> If it is not set, a random password is generated. It is recommended to reset the admin password after installation.|
+|everest|`server.oidc`|object|{}|OIDC configuration for Everest.</br></br> These settings are applied only during installation. To modify the settings after installation, you have to manually update the everest-settings `ConfigMap`.|
+
+|**Subcharts**|**Key**|**Type**|**Default**|**Description**|
+|---------|------|---------|------------------|------------|
+|everest-db-namespace|`pxc`|bool|true| Installs the Percona XtraDB Cluster operator if set.|
+|everest-db-namespace|`postgresql`|bool|true| Installs the Percona Postgresql Server operator if set.|
+|everest-db-namespace|`psmdb`|bool|true| Installs the Percona Server MongoDB operator if set.|
```

Comment:
If you're creating 2 separate tables, the `Charts` and `Subcharts` columns are not needed

```suggestion
For `percona/everest` chart:
|**Key**|**Type**|**Default**|**Description**|
|------|---------|------------------|------------|
|`server.initialAdminPassword`|string|""|Initial password configured for admin user.</br></br> If it is not set, a random password is generated. It is recommended to reset the admin password after installation.|
|`server.oidc`|object|{}|OIDC configuration for Everest.</br></br> These settings are applied only during installation. To modify the settings after installation, you have to manually update the everest-settings `ConfigMap`.|
 
For `percona/everest-db-namespace` subchart
|**Key**|**Type**|**Default**|**Description**|
|------|---------|------------------|------------|
|`pxc`|bool|true| Installs the Percona XtraDB Cluster operator if set.|
|`postgresql`|bool|true| Installs the Percona Postgresql Server operator if set.|
|`psmdb`|bool|true| Installs the Percona Server MongoDB operator if set.|
```